airbyte-ci/connectors/metadata_service/lib/metadata_service/models/generated/ConnectorRegistryDestinationDefinition.py

```diff
@@ -255,7 +255,7 @@ class ConnectorReleaseCandidates(BaseModel):
     class Config:
         extra = Extra.forbid
 
-    __root__: Dict[constr(regex=r"^\d+\.\d+\.\d+$"), VersionReleaseCandidate] = Field(
+    __root__: Dict[constr(regex=r"^\d+\.\d+\.\d+(-[0-9A-Za-z-.]+)?$"), VersionReleaseCandidate] = Field(
```

Comment:
Any reason not to use something stricter, e.g. `^\d+\.\d+\.\d+(-rc\.[0-9]+)$`?

Reply:
If we want to be stricter I think it should be at the metadata validation side, which is what we've today. If we're stricter here it might lead to blocking registry generation error.